js/tests/enhancedDate.spec.js

```diff
@@ -0,0 +1,90 @@
+//  Patrice Briggs
+//  Homework Module 6
+
+//test suite for enhancedDate function
+describe("enhancedDate", function () {    
+    'use strict';
+
+    //test setDate function
+    describe("setDate", function () {
+        it("should return a new Date Object if nothing passed in", function () {
+        	expect(enhancedDate.setDate()).to.be.instanceOf(new Date());
```

Comment:
instanceof just checks against the object constructor, not an object instance:

`expect(enhancedDate.setDate()).to.be.instanceof(Date);`